elasticsearch.md

```diff
@@ -912,55 +913,113 @@ readonlyrest:
 
-#### `kibana_index`
-
-`kibana_index: .kibana-user1`
+##### `index` 
 
 **Default value is `.kibana`**
 
-Specify to what index we expect Kibana to attempt to read/write its settings \(use this together with `kibana.index` setting in kibana.yml.\)
+Specify to what index we expect Kibana to attempt to read/write its settings (use this together with `kibana.index` setting in the `kibana.yml` file)
 
-This value directly affects how `kibana_access` works because at all the access levels \(yes, even admin\), `kibana_access` rule will **not** match any _write_ request in indices that are not the designated kibana index.
+This value directly affects how `kibana.access` works because at all the access levels \(yes, even admin\), `kibana.access` sub-rule will **NOT** match any _write_ request in indices that are not the designated kibana index.
 
 If used in conjunction with ReadonlyREST Enterprise, this rule enables **multi tenancy**, because in ReadonlyREST, a tenancy is identified with a set of Kibana configurations, which are by design collected inside a kibana index \(default: `.kibana`\).
 
-#### `kibana_index_template`
+It supports [dynamic variables](./elasticsearch.md#dynamic-variables).
 
-`kibana_index_template: .kibana_template`
+##### `template_index`
 
 Used to pre-populate tenancies with default kibana objects, like dashboards and visualizations. Thus providing a starting point for new tenants that will avoid the bad user experience of logging for the first time and finding a completely empty Kibana.
 
-#### `kibana_hide_apps`
+It supports [dynamic variables](./elasticsearch.md#dynamic-variables).
 
-`kibana_hide_apps: [ "Security", "Enterprise Search"]`
+##### `hide_apps`
 
 Specify which Kibana apps and menu items should be hidden. 
 This feature will work in ReadonlyREST PRO and Enterprise.
 
 For more information on the ROR's Kibana Hide Apps feature, see [Hiding Kibana Apps](kibana.md#hiding-kibana-apps).
 
+##### `allowed_api_paths`
+
+Used to define which parts of [Kibana REST API](https://www.elastic.co/guide/en/kibana/current/api.html) can be used. The sub-rule requires to define a list of [regular expressions](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html) which describes the API paths. Additionally, when you would like to restrict only specific HTTP methods of the API, you can use the extended format of the sub-rule:
+
+```yaml
+kibana:
+  [...]
+  allowed_api_paths: # optional
+    - http_method: POST
+      path: "^/api/saved_objects/.*$"
+```
+
+##### `metadata`
+
+User to define the Custom ROR Kibana Metadata which can be used in [Custom middleware](./kibana.md#custom-middleware). The `kibana.metadata` in ReadonlyREST settings is an unstructured YAML object. 
+
+It supports [dynamic variables](./elasticsearch.md#dynamic-variables).
```

Comment:
Great, let's show off the power of using dynamic variables in metadata. 

Let's give them some examples, maybe in a different page (together with more example we can add in the future). 

I.e.

```yml
metadata:
    alert_message:  "Dear @{acl.current_group} users, you are viewing dashboards for indices @explode{acl:available_groups}_logstash-*"
    
```

@Dzuming let's put here a tiny, simple html+js snippet they can put in kibana.yml that looks for the metadata field `alert_message` and when it finds a valid string goes:

```
// Message from security admin
alert(alert_message)
```

Reply:
Added the simple snippet ✔️